src/providers/consensus/client.py

```diff
@@ -151,6 +152,25 @@ def get_state_block_roots(self, state_id: SlotNumber) -> list[BlockRoot]:
             ))
         return list(streamed_json['data']['block_roots'])
 
+    @lru_cache(maxsize=1)
+    def get_state_view(self, state_id: SlotNumber | StateRoot) -> BeaconStateView:
+        """Spec: https://ethereum.github.io/beacon-APIs/#/Debug/getStateV2"""
+        streamed_json = cast(TransientStreamingJSONObject, self._get(
+                self.API_GET_STATE,
+                path_params=(state_id,),
+                stream=True,
+            ))
+        view = {}
+        data = streamed_json['data']
+        try:
+            # NOTE: Keep in mind: the order is important, see TransientStreamingJSONObject.
+            view['slot'] = int(data['slot'])
```

Comment:
Are we sure that keys in json will be always in next order?

Reply:
In general, I don't know since JSON has no ordering. But all the tests are fine so far, and I've never seen the order changed so far.